Removed empty cell

Mahesh1822 opened the pull request:
### Pull Request Description
Removed empty cell from docs/source/demos/text_input.ipynb  "Fixes #3206"

jeremyliweishih commented on docs/source/release_notes.rst:
```diff
@@ -7,6 +7,7 @@ Release Notes
         * Fixed classification pipelines to only accept target data with the appropriate number of classes :pr:`3185`
         * Added support for time series in ``DefaultAlgorithm`` :pr:`3177`
         * Standardized names of featurization components :pr:`3192`
+	* Removed empty cell :pr:`3234`
```
Could you add some more color here? Something like, "Removed empty cell in text_input.ipynb  :pr:`3234`" would be great!

jeremyliweishih commented:
Thanks for putting up this new PR @Mahesh1822, just had one comment and I can approve after. In the future you do not need to put up a new PR! You can just make changes to the branch you're on, push the branch up, and the change will be in your PR as well. Let me know if that makes sense and how else I can help. Since a commit has been merged into our `main` (base) branch, you will need to update this branch by merging in the latest changes from main as well before you can merge this PR.

jeremyliweishih commented:
Another handy thing to note is: if you add "Fixes #3206" in your PR description, it will automatically close that issue once this PR goes in. Heres some documentation from github on how that works: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue!

Mahesh1822 commented:
Thanks I did the changes and updated my repo too. Please check if everything is right.

jeremyliweishih reviewed (approved):
Thank you this is great! Should be good once tests pass!

Mahesh1822 commented:
It's showing a conflict. Why is it?

Mahesh1822 commented:
`<<<<<<< remove_cell
	* Removed empty cell in text_input.ipynb :pr:`3234`
=======
        * Removed potential prediction explanations failure when pipelines predicted a class with probability 1 :pr:`3221`
>>>>>>> main`

Mahesh1822 commented:
<<<<<<< remove_cell
	* Removed empty cell in text_input.ipynb :pr:`3234`
=======
        * Removed potential prediction explanations failure when pipelines predicted a class with probability 1 :pr:`3221`
>>>>>>> main

Mahesh1822 commented:
I edited the doc a bit. And it solved the conflict.

jeremyliweishih commented:
@Mahesh1822 in general a conflict arises if someone else merged a change into `main` that touches the files in this PR. I didn't see the exact conflict but I imagine it was because someone made a change in main and had a change in the release notes that coincided with yours. Thanks for fixing the conflict!

chukarsten commented:
Thanks for contributing @Mahesh1822 !

jeremyliweishih commented:
@Mahesh1822 I'm going to merge before another conflict arises. Thank you!

Mahesh1822 commented:
Here goes my first contribution : )
